Cleanups

kamuikatsurgi opened the pull request:
- Fixes `--monitor` to correctly include `lastStateIDFromBor`.
- Some minor CI and anvil setup cleanups to make the code cleaner and more sociable. (Mostly removing unnecessary lines and stuff).

marcello33 commented:
LGTM.
2 minor things:
- Can we preserve the `fund_anvil_account` script in the docker util folder? Just in case we want to run it for testing. However we can delete it from the `.github` folder now, it should not be needed, right?
- Since you're here, can you remove the `fund_anvil_account` script execution from the `--start` command? If not needed, it can be removed also from the `--cleanup` command.
Thanks

kamuikatsurgi commented:
>Can we preserve the `fund_anvil_account` script in the docker util folder?

I deleted the `fund_ganache_account` script; `fund_anvil_account` is still there.

>However we can delete it from the .github folder now, it should not be needed, right?

Removed `fund_anvil_account` from `.github` folder.

> can you remove the fund_anvil_account script execution from the --start command? If not needed, it can be removed also from the --cleanup command.

Removed `fundAnvilAccouts` from `--start` and `--cleanup`.
